www/js/app.js

```diff
@@ -50,6 +50,8 @@ var app = {
     // The scope of 'this' is the event. In order to call the 'receivedEvent'
     // function, we must explicitly call 'app.receivedEvent(...);'
     onDeviceReady: function() {
+        initializeCrittercism();
```

Comment:
Can we do it in 4 lines right here?  That'll make it an easy copy/paste for customers.
Also, I think "device.platform" is what you want (http://docs.phonegap.com/en/edge/cordova_device_device.md.html#device.platform).

    if (device.platform === "iOS") {
        Crittercism.init("YOUR IOS APP ID");
    } else if (device.platform === "Android") {
        Crittercism.init("YOUR ANDROID APP ID");
    }